app/Http/Controllers/Licenses/LicenseCheckinController.php

```diff
@@ -106,4 +106,44 @@ public function store(Request $request, $seatId = null, $backTo = null)
         // Redirect to the license page with error
         return redirect()->route('licenses.index')->with('error', trans('admin/licenses/message.checkin.error'));
     }
+
+    public function checkinCount($license_id){
+        $licenseSeats = LicenseSeat::where('license_id', '=', $license_id)
+            ->whereNotNull('assigned_to')
+            ->with('user')
+            ->get();
+
+        return('There are '.$licenseSeats->count().' seats checked out. Are you sure you want to check all of them in?');
+    }
+
+
+    public static function checkinLicense($license_id){
+
+        $licenseSeats = LicenseSeat::where('license_id', '=', $license_id)
+            ->whereNotNull('assigned_to')
+            ->with('user')
+            ->get();
+
+        if (! $license_id) {
+            return redirect()->to('/licenses')->with('error', 'License does not exist.');
+
+        }
+
+        if (!License::where('id', '=', $license_id)->first()) {
+            return redirect()->to('/licenses')->with('error', 'Invalid license ID.');
```

Comment:
This statement seems redundant to me? If the license doesn't exist, it doesn't exist. The if statement above would catch it, no?

Reply:
It does, I removed one.